src/styles/header.scss

```diff
@@ -0,0 +1,62 @@
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px ;
+  &__logo {
+    margin-top: 10px;
+    width: 40px;
+    height: 40px;
+    &-img {
+      width: 100%;
+      height: 100%;
+    }
+  }
+}
+  
+.nav {
+  padding: 10px 0 20px;
+  &__list {
+    margin-top: 0;
+    margin-bottom: 4px;
+    padding-top: 9px;
+    padding-left: 0;
+    list-style: none;
+    display: flex;
+  }
+  &__item {
+    position: relative;
+    margin-left: 20px;
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+  &__link {
+    color: $main-color-text;
+    font-size: 12px;
+    text-decoration: none;
+    font-weight: 500;
+    text-transform: uppercase;
+    padding: 23px 0;
+    &:hover {
+      color: #00acdc;
+    }
+  }
+
+  .is-active {
+    color: #00acdc;
+    &::after {
```

Comment:
if you use same color a few times, maybe it would be great to move it into variable